core-web/.husky/pre-commit

```diff
@@ -264,6 +263,21 @@ fi
 # Check if sdk-client is affected
 check_sdk_client_affected
 
+# Check if there are changes in core-web/libs/sdk and run bump-sdk-versions.js if needed
+sdk_files_staged=$(echo "$staged_files" | grep -E '^core-web/libs/sdk/' || true)
+if [ -n "$sdk_files_staged" ]; then
+    echo "Changes detected in core-web/libs/sdk. Running bump-sdk-versions.js"
+    if ! node "${root_dir}/core-web/bump-sdk-versions.js"; then
```

Comment:
This will use whatever node is in the environment path.  We need to ensure we use the node in the projects installs/node/node that is put there by the maven frontend plugin process.   we can set the path the same way we do for yarn,  alternatively up front in the script we could put the correct yarn and node in the PATH so it is picked up in preference to any other to prevent issues like this going forward.

Reply:
The function `load_nvm_and_use_node()` was implemented in order to load `nvm` and use the version specified in `.nvmrc` file if present.